src/main/java/com/opendemobank/backend/controller/TransactionController.java

```diff
@@ -0,0 +1,79 @@
+package com.opendemobank.backend.controller;
+
+import com.opendemobank.backend.domain.*;
+import com.opendemobank.backend.repository.TransactionsRepo;
+import com.opendemobank.backend.repository.UsersRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Objects;
+
+@RestController
+@RequestMapping("/api/transactions")
+public class TransactionController {
+
+    @Autowired
+    TransactionsRepo transactionsRepo;
+
+    @Autowired
+    UsersRepo usersRepo;
+
+    @GetMapping("/{id}")
+    public ResponseEntity<Transaction> getById(@AuthenticationPrincipal User currentUser, @PathVariable("id") long id) {
```

Comment:
Add 
`@Parameter(hidden = true) `
to currentUser so api documentation doesnt show it.

eg. 
`@Parameter(hidden = true) @AuthenticationPrincipal final User currentUser`